### PR TITLE
Fix: PHP notice trying to access array offset on value of type null.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Issue found while testing a some bugs in next-release ( 2.6.9 ).

### Screenshots

- https://share.getcloudapp.com/nOuLnJLg

### Types of changes

- Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?

- Screencast added for testing.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: blocks-config/post/class-uagb-post.php [hunk 1]
```diff
@@ -1830,11 +1830,14 @@ public function add_post_dynamic_script() {
 				}
 			}
 
-			if ( is_array( self::$settings['grid'] ) && ! empty( self::$settings['grid'] ) ) {
+			if ( isset( self::$settings['grid'] ) && is_array( self::$settings['grid'] ) && ! empty( self::$settings['grid'] ) ) {
```
Comment (adi3890) [inline blocks-config/post/class-uagb-post.php:1833]: instead of adding isset if we bring ! empty before this should work
I feel this is about the order of logic.

Ideall empty check should be first thing

File: blocks-config/post/class-uagb-post.php [hunk 2]
```diff
@@ -1830,11 +1830,14 @@ public function add_post_dynamic_script() {
...
 				foreach ( self::$settings['grid'] as $key => $value ) {
 					if ( 'ajax' !== $value['paginationType'] ) { 
 						return; // Early return when pagination type is not ajax.
 					}
+					if ( ! isset( $key ) ) {
```
Comment (adi3890) [inline blocks-config/post/class-uagb-post.php:1838]: any specific reason for this check

File: blocks-config/post/class-uagb-post.php (hunk 1 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: Right, checking `! empty` will resolve the issue

File: blocks-config/post/class-uagb-post.php (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/post/class-uagb-post.php]: No, Its not needed anymore.

**Review by adi3890 — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Comment by mohitsbsftester** [comment]
Good to go.
